Goa developer offline

greenageinfo opened the pull request.

greenageinfo committed:
Update _config.yml

changed config baseurl to ''

greenageinfo committed:
Update _config.yml

Changed to site details and logo

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Update _config.yml

customization of site title to Greenage

greenageinfo committed:
Update _config.yml

changes to title

greenageinfo committed:
Update _config.yml

updated logo

greenageinfo committed:
Update _config.yml

changes to authors

greenageinfo committed:
Update default.html

updates to top menu nav

greenageinfo committed:
Update about.md

updated content

greenageinfo committed:
Add files via upload

added greenage layout

greenageinfo committed:
Update default.html

nav updates

greenageinfo committed:
Update about.md

Changes to content

greenageinfo committed:
Update _config.yml

changed link for logo

greenageinfo committed:
Update about.md

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Delete logo.png

greenageinfo committed:
Add files via upload

uploaded logo and layout

greenageinfo committed:
Update about.md

updates to links

greenageinfo committed:
Update about.md

greenageinfo committed:
Update 2018-01-11-quick-start-guide.md

greenageinfo committed:
Update and rename 2018-01-11-quick-start-guide.md to 2020-08-14-quick-start-guide.md

greenageinfo committed:
Update 2020-08-14-quick-start-guide.md

greenageinfo committed:
Update 2020-08-14-quick-start-guide.md

greenageinfo committed:
Update 2020-08-14-quick-start-guide.md

greenageinfo committed:
Delete 2018-01-11-customer-service.md

greenageinfo committed:
Delete 2018-01-12-is-intelligence-enough.md

Commit:
deleted few posts

deleted few posts

greenageinfo committed:
Create 2020-08-14-about-greenage.md

new post

greenageinfo committed:
Update about.md

greenageinfo committed:
Update about.md

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Update default.html

greenageinfo committed:
Add files via upload

uploaded images

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-quick-start-guide.md

greenageinfo committed:
Add files via upload

images

greenageinfo committed:
Update 2020-08-14-about-greenage.md

greenageinfo committed:
Update 2020-08-14-quick-start-guide.md

greenageinfo committed:
Update _config.yml

update to baseurl

greenageinfo committed:
Update _config.yml

greenageinfo committed:
Update post.html

Commit:
updates to site

updates

Commit:
Update 2020-08-16-greenage-visitor.md

changes to content

Commit:
Update 2020-08-16-greenage-visitor.md

minor edit

Commit:
Create contacts.md

add contacts list

greenageinfo committed:
Update default.html

added nav

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update default.html

add table script

greenageinfo committed:
Update default.html

update

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update default.html

add data table script

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update default.html

greenageinfo committed:
Update default.html

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update default.html

greenageinfo committed:
Update contacts.md

Commit:
Create contacts.yaml

new data file

greenageinfo committed:
Update and rename contacts.yaml to contacts.yml

greenageinfo committed:
Update contacts.md

yml integration

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.yml

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update default.html

add tables css

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update default.html

Commit:
Create table-style.css

Commit:
Merge branch 'GOA-developer-offline' of https://github.com/greenageinfo/greenageinfo.github.io into GOA-developer-offline

greenageinfo committed:
Update default.html

greenageinfo committed:
Update default.html

Commit:
Create table.style.min.css

greenageinfo committed:
Update default.html

greenageinfo committed:
Update default.html

greenageinfo committed:
Update 2020-08-16-greenage-visitor.md

greenageinfo committed:
Add files via upload

greenageinfo committed:
Update contacts.md

greenageinfo committed:
Update 2020-08-14-quick-start-guide.md

Commit:
logo changes

logo changes

Commit:
Merge branch 'GOA-developer-offline' of https://github.com/greenageinfo/greenageinfo.github.io into GOA-developer-offline

Commit:
Revamped website

changes

Commit:
summary

Commit:
Changes across the site

Changes across the site

greenageinfo committed:
Merge pull request #1 from greenageinfo/GOA-developer-offline

Merge Changes to Main Branch

Commit:
change main author name

greenageinfo committed:
Merge pull request #2 from greenageinfo/GOA-developer-offline

change main author name

greenageinfo committed:
Create CNAME

jasonsdaddy commented on _config.yml:
```diff
@@ -2,8 +2,8 @@
+favicon: 'assets/images/greenage-logo.png'
 baseurl: ''
 google_analytics: 'UA-46278016-1'
 disqus: 'demowebsite'
```
#156 #
Duplicate of #156 #155 @

>

jasonsdaddy reviewed (approved):
[Jasmyne Marie Allen ](jasmynemarie94@icloud.com)

Commit:
Update 2020-08-26-interiors.md

link update

greenageinfo committed:
Merge pull request #3 from greenageinfo/GOA-developer-offline

Update 2020-08-26-interiors.md

Commit:
update

Commit:
Create d84bf72b90ab41d1faece503994e50e82580faea7fa954a0dfed93b2b88d4f

greenageinfo committed:
Merge pull request #4 from greenageinfo/GOA-developer-offline

Goa developer offline

Commit:
update to handbook link

greenageinfo committed:
Merge pull request #5 from greenageinfo/GOA-developer-offline

update to handbook link

Commit:
Updates to Contacts file

Updates to Contacts file

greenageinfo committed:
Merge pull request #6 from greenageinfo/GOA-developer-offline

Updates to Contacts file

Commit:
changes to contacts

changes to contacts

greenageinfo committed:
Merge pull request #7 from greenageinfo/GOA-developer-offline

changes to contacts

Commit:
Changes to pets page,

Changes to pets page, move-in and move-out changes, secondary sale of properties name transfer

greenageinfo committed:
Merge pull request #8 from greenageinfo/GOA-developer-offline

Changes to pets page,

Commit:
covid page changes

covid page changes

greenageinfo committed:
Merge pull request #9 from greenageinfo/GOA-developer-offline

covid page changes

Commit:
change to covid page

change to covid page

greenageinfo committed:
Merge pull request #10 from greenageinfo/GOA-developer-offline

change to covid page

Commit:
Update 2020-08-26-interiors.md

changes to interior page

greenageinfo committed:
Merge pull request #11 from greenageinfo/GOA-developer-offline

Update 2020-08-26-interiors.md

Commit:
new page added

new page added

greenageinfo committed:
Merge pull request #12 from greenageinfo/GOA-developer-offline

new page added

Commit:
updated contacts page

greenageinfo committed:
Merge pull request #13 from greenageinfo/GOA-developer-offline

updated contacts page

Commit:
segregation page

greenageinfo committed:
Merge pull request #14 from greenageinfo/GOA-developer-offline

segregation page

Commit:
added image

greenageinfo committed:
Merge pull request #15 from greenageinfo/GOA-developer-offline

added image

Commit:
change in cover pic

greenageinfo committed:
Merge pull request #16 from greenageinfo/GOA-developer-offline

change in cover pic

Commit:
covid page updates

greenageinfo commented:
OK

greenageinfo committed:
Merge pull request #17 from greenageinfo/GOA-developer-offline

covid page updates

Commit:
updated covid

greenageinfo committed:
Merge pull request #18 from greenageinfo/GOA-developer-offline

updated covid

Commit:
changes to segregation page

greenageinfo committed:
Merge pull request #19 from greenageinfo/GOA-developer-offline

changes to segregation page

Commit:
Update 2020-07-15-segregation.md

Commit:
Update 2020-07-15-segregation.md

greenageinfo committed:
Merge pull request #20 from greenageinfo/GOA-developer-offline

Goa developer offline

Commit:
updates to coivd page

greenageinfo committed:
Merge pull request #21 from greenageinfo/GOA-developer-offline

updates to coivd page

Commit:
changes to multiple pages

greenageinfo committed:
Merge pull request #22 from greenageinfo/GOA-developer-offline

changes to multiple pages

vvrkris committed:
changes to waste seggregation page

greenageinfo committed:
Merge pull request #23 from greenageinfo/GOA-developer-offline

changes to waste seggregation page

vvrkris committed:
added google analytics

greenageinfo committed:
Merge pull request #24 from greenageinfo/GOA-developer-offline

added google analytics

vvrkris committed:
covid page updated links

greenageinfo committed:
Merge pull request #25 from greenageinfo/GOA-developer-offline

covid page updated links

vvrkris committed:
changes to multiple pages

greenageinfo committed:
Merge pull request #26 from greenageinfo/GOA-developer-offline

changes to multiple pages

vvrkris committed:
new gallery page. updates to shops page. New greenage cares post added

greenageinfo committed:
Merge pull request #27 from greenageinfo/GOA-developer-offline

new gallery page. updates to shops page. New greenage cares post added

vvrkris committed:
changes to covid page

greenageinfo committed:
Merge pull request #28 from greenageinfo/GOA-developer-offline

changes to covid page

vvrkris committed:
changes done

greenageinfo committed:
Merge pull request #29 from greenageinfo/GOA-developer-offline

changes done

greenageinfo committed:
Update CNAME

greenageinfo committed:
Update CNAME

greenageinfo committed:
Update CNAME

vvrkris committed:
changes

greenageinfo committed:
Merge pull request #30 from greenageinfo/GOA-developer-offline

changes

vvrkris committed:
minor formatting changes

greenageinfo committed:
Merge pull request #31 from greenageinfo/GOA-developer-offline

minor formatting changes

vvrkris committed:
minor formatting changes

greenageinfo committed:
Merge pull request #32 from greenageinfo/GOA-developer-offline

minor formatting changes

brajeshwar committed:
Internet in Greenage

greenageinfo committed:
Update 2022-01-01-internet.md

greenageinfo committed:
Merge pull request #33 from greenageinfo/post/internet

Internet and DTH Services in Greenage

Commit:
minor changes

Commit:
pull detals

Commit:
minor changes to internet page

Commit:
change to the image

Commit:
modified the pic

Commit:
all changes

Commit:
changes to the sinx file

Commit:
commit

greenageinfo committed:
Merge pull request #34 from greenageinfo/GOA-developer-offline

Goa developer offline

Commit:
changes to New page. added link to internet

Commit:
changes to new page

greenageinfo committed:
Merge branch 'master' into GOA-developer-offline

vvrkris committed:
sync

vvrkris committed:
changes to covid page

Commit:
download latest

Commit:
commmit

Commit:
commit

Commit:
final commit

Commit:
change to dues page penalties link

Commit:
newsletter jan 22 page

Commit:
changes to newsletter page

Commit:
changes to newsletter

Commit:
changes to newsletter page

vvrkris committed:
updates to newsletter page

Commit:
new aminities post

vvrkris committed:
new page - flat lekage policy

vvrkris committed:
final commit

vvrkris committed:
changes to order of pages

vvrkris committed:
changes to secondary property sales process

vvrkris committed:
changes to payments page

vvrkris committed:
changed to interiors pages

vvrkris committed:
interiors changes

vvrkris committed:
changes to contact page

vvrkris committed:
changes to contact page

greenageinfo committed:
Update contacts.md

vvrkris committed:
updated contacts page

vvrkris committed:
new page for promotions

vvrkris committed:
newsletter for april

vvrkris committed:
change to COC link

vvrkris committed:
new library page first draft

Commit:
changes to library page

vvrkris committed:
changes to library page

vvrkris committed:
changes to library page

vvrkris committed:
change to number of books. library

vvrkris committed:
updates to library page

vvrkris committed:
final library page as of 5.Jun
